README.md

```diff
@@ -112,7 +112,8 @@ altering some of the default settings.
   xenial, bionic and default), `false` for Debian (wheezy and default)):
   Use the legacy `Unattended-Upgrade::Allowed-Origins` setting or the modern `Unattended-Upgrade::Origins-Pattern`.
 * `mail`: A hash to configure email behaviour with two possible keys:
```

Comment:
‘Two possible keys’?

Reply:
Ah yeah, there are now three possible keys. #183 fixes this.